build/onion-editor.js

```diff
@@ -11621,12 +11621,21 @@ define('onion-editor',[
     scribe.use(scribePluginCurlyQuotes());
     scribe.use(scribePluginKeyboardShortcuts(Object.freeze(keyCommands)));
 
-    //TODO: kill this existing toolbar & replace w/ Medium style selection toolbar
+    // garbage toolbar setup
```

Comment:
This comment does not document anything but your feelings on the setup, please remove it.